crates/matrix-sdk-crypto/src/olm/group_sessions/sender_data.rs

```diff
@@ -71,12 +71,30 @@ where
             A: de::SeqAccess<'de>,
         {
             let mut buf = [0u8; 32];
+
             for (i, item) in buf.iter_mut().enumerate() {
                 *item = seq.next_element()?.ok_or_else(|| de::Error::invalid_length(i, &self))?;
             }
+
             let key = Ed25519PublicKey::from_slice(&buf).map_err(|e| de::Error::custom(&e))?;
+
             Ok(Box::new(key))
         }
+
+        fn visit_bytes<E>(self, v: &[u8]) -> Result<Self::Value, E>
```

Comment:
Any tips on how to confirm that `visit_bytes` was the only thing we missed? (I'm not very familiar with these bits of serde)

Reply:
1. Tell rust-analyzer to implement all the default members.
2. See that all the relevant other types (`Vec<u8>`, `String`, `&'de [u8]`) call `visit_bytes()` or `visit_str()` by default.

![image](https://github.com/user-attachments/assets/791e876f-45c8-44a3-8349-6a5cfacedd69)

You could also read the documentation: https://rust-lang.github.io/hashbrown/serde/de/trait.Visitor.html#method.visit_byte_buf. :nerd_face:

Reply:
Perfect, thank you.